components/Tabs/index.js

```diff
@@ -7,3 +7,19 @@
 //
 //  The tab component should look like this:
 //    <div class="tab">topic here</div>
+
+axios.get('https://lambda-times-backend.herokuapp.com/topics')
+
+.then(info => info.data.topics.forEach(element => new Tab(element)))
+
```

Comment:
It is always a good practice to chain a .catch function to handle errors when using promises